robots/anymal_b_simple_description/robots/anymal.urdf

```diff
@@ -101,25 +101,24 @@
       </geometry>
     </collision>
     <inertial>
-      <origin rpy="0 0 0" xyz="-0.001960558279 -0.001413217745 0.050207125344"/>
-      <mass value="16.793507758"/>
-      <inertia ixx="0.217391101503" ixy="-0.00132873239126" ixz="-0.00228200226173" iyy="0.639432546734" iyz="-0.00138078263145" izz="0.62414077654"/>
+      <mass value="1e-6"/>
```

Comment:
I don’t really like this trick. It also mean that we cannot trust the URDF file anymore when used in external applications. Can you motivate this change?

Reply:
Gazebo used to require this trick, but to be honest, I don't know exactly the reason. I always assume that it is something related to ODE optimization. @wxmerkt is this still valid? could we remove this trick?

Btw, I have tried in Pybullet, and we don't need this trick.
Indeed, I am confused. I though that Pybullet requires the same trick.
@wxmerkt what was the motivation of your previous change?

Reply:
PyBullet assumes a 1kg mass and ixx=iyy=izz=1 inertia if no value is given, so unless you can find a reference of a recent change in PyBullet (pull request), I'd say we do need it

Reply:
Thanks for the clarification.

Just curiosity, what about Gazebo?

Reply:
I don't know about Gazebo in its latest versions but I believe it's fine without this work-around now. RBDL would require small inertias as well otherwise it segfaults on loading.

Reply:
For completeness, ICub and Talos don't have this trick; instead HyQ has it.

I could handle it in this PR. @wxmerkt, @jcarpent  do you agree with it?

Reply:
I only used HyQ and Anymal during MPC development so that's why the other two dont have it. Happy for you to add it either now or when it's required/validated

Reply:
I corrected myself, Talos doesn't need this trick.

I have pushed the modifications of the remaining robots.